back/test/auth.py

```diff
@@ -115,6 +115,7 @@ def create_jwt_payload(
             "shipment:edit",
             "transfer_agreement:create",
             "transfer_agreement:edit",
+            "transfer_agreement_detail:read",
```

Comment:
Did you update this in auth0?

Reply:
It's already present in Auth0, and I'm adding it here to mock the JWT content that Auth0 would create